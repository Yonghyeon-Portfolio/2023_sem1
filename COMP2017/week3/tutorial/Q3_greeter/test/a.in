personA 25 Apple
personB 23 Beer
personC 53 Candle
personD 23 donkey
personE 22 bass-drum
personF 5 hello
personG 25 world
personH 21 gate
personI 22 hat
personJ 35 fish
personK 45 holy
personL 73 moly
personM 63 hi
personN 50 yoga
personO 50 yoyo
