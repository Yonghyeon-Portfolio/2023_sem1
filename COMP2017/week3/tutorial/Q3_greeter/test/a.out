Welcome to ShopaMocha,
Customer  0, Name: personA, Age: 25, Looking for: Apple
Customer  1, Name: personB, Age: 23, Looking for: Beer
Customer  2, Name: personC, Age: 53, Looking for: Candle
Customer  3, Name: personD, Age: 23, Looking for: donkey
Customer  4, Name: personE, Age: 22, Looking for: bass-drum
Customer  5, Name: personF, Age: 5, Looking for: hello
Customer  6, Name: personG, Age: 25, Looking for: world
Customer  7, Name: personH, Age: 21, Looking for: gate
Customer  8, Name: personI, Age: 22, Looking for: hat
Customer  9, Name: personJ, Age: 35, Looking for: fish
Customer 10, Name: personK, Age: 45, Looking for: holy
Customer 11, Name: personL, Age: 73, Looking for: moly
Customer 12, Name: personM, Age: 63, Looking for: hi
Customer 13, Name: personN, Age: 50, Looking for: yoga
Customer 14, Name: personO, Age: 50, Looking for: yoyo
